libs/pages/elewa/social-impact/src/lib/components/social-impact-beyond-business/social-impact-beyond-business.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'elewa-group-social-impact-beyond-business',
+  templateUrl: './social-impact-beyond-business.component.html',
+  styleUrls: ['./social-impact-beyond-business.component.scss'],
+})
+export class SocialImpactBeyondBusinessComponent {
+@Input() paragraphs = ["Some of that youthful energy is channeled beyond business. Through projects such as our coding school in Kakuma refugee camp, training teachers in pastoral Samburu and our open-source coding initiatives providing entry-level opportunities to hundreds of junior engineers.", "These projects don't necessarily have an immediate financial return, but their impact can be tangibly measured in the long-term."]
```

Comment:
This should just be a property named paragraphs, no need for the Input decorator

Reply:
Hi @LemmyMwaura, I have changed the paragraph into a property and pushed the changes

Reply:
Okay, Just sync the latest updates to resolve the conflicts